channels/chan_usbradio.c

```diff
@@ -4388,6 +4388,36 @@ static void tune_rxctcss(int fd, struct chan_usbradio_pvt *o, int intflag)
 	o->pmrChan->b.tuning = 0;
 }
 
+/*!
+ * \brief Update the tune settings to the configuration file.
+ * \param filename	The configuration file being updated (e.g. "usbradio.conf").
+ * \param category	The category being updated (e.g. "12345").
+ * \param variable	The variable being updated (e.g. "rxboost").
+ * \param value		The value being updated (e.g. "yes").
+ */
+static int tune_variable_update(const char *filename, struct ast_category *category,
+								const char *variable, const char *value)
+{
+	int	res;
```

Comment:
should just be one space between int and res

Reply:
That was one <tab>.  Changed to a <space>